Formula/cjdns.rb

```diff
@@ -20,6 +20,40 @@ def install
     (pkgshare/"test").install "build_darwin/test_testcjdroute_c" => "cjdroute_test"
   end
 
+  def post_install
+    `#{bin}/cjdroute --genconf > #{HOMEBREW_PREFIX}/etc/cjdroute.conf`
```

Comment:
This should be done in `install` and
```
(buildpath/"cjdroute.conf").write Utils.popen_read(bin/"cjdroute", "--genconf")
etc.install "cjdroute.conf"
```

Reply:
Unfortunately it doesn't work.

Reply:
They're equivalent. So you need to elaborate.

Reply:
When using solution from your comment `cjdroute.conf` file is missing, I don't know what else to tell you.

Reply:
```
🍺  /usr/local/Cellar/cjdns/20.1: 7 files, 1.7MB, built in 24 seconds
iMac-TMP:~ joe$ ls /usr/local/etc/cjdroute.conf 
/usr/local/etc/cjdroute.conf
```

Reply:
```
🍺  /usr/local/Cellar/cjdns/20.1: 7 files, 1.7MB
~ ❯ ls /usr/local/etc/cjdroute.conf
ls: /usr/local/etc/cjdroute.conf: No such file or directory
```

Reply:
If I'm not mistaken `install` step is skipped when formula is poured. That's the reason I've used `post_install` hook.

Reply:
Yes, you are mistaken.

Reply:
So why `install` method is not invoked then (and hence missing `cjdroute.conf`)?

Reply:
The files that have been installed in etc while the bottle was built are collected in the bottles's hidden .bottle directory and installed from there even though the `install` method itself does not run when pouring.

Reply:
So how can I generate config file, on the fly, if `install` method is **not** being run?

Reply:
You need to trust me on how this works. Unfortunately, I don't have time to elaborate beyond what I said above.

Reply:
btw, you've said I'm mistaken, but in the very next message you proved me right by saying:

> [...] even though the install method itself **does not run when pouring**.